src/quest_manager/models.py

```diff
@@ -975,3 +988,55 @@ def get_minutes_to_complete(self):
 
         minutes = (self.first_time_completed - self.timestamp).total_seconds() / 60
         return minutes
+
+
+class QuestionType(models.Model):
+    id = models.CharField(
+        max_length=100,
+        primary_key=True,
+        help_text="Short, memorable, unique identifier for a question type e.g. 'short_answer', 'file_upload', 'multiple_choice'.",
+        serialize=False
+    )
+    name = models.CharField(
+        max_length=255,
+        help_text="The name of the question type e.g. 'Short Answer', 'File Upload', 'Multiple Choice'."
+    )
+    description = models.TextField(blank=True, null=True)
+
+    def __str__(self):
+        return self.name
+
+
+class Question(models.Model):
+    instructions = models.TextField()
```

Comment:
All of these fields should have help_text.  Since these will be user facing models, we want good help text (which by default is the help text that appears in forms) to make use of this feature as easy as possible.

Reply:
I added help_text for all user-facing fields. I didn't add it for some fields, like the foreignkeys, because I wasn't sure if it would be helpful. Let me know if I should add help_text for them too.
```python
class Question(models.Model):
...
    quest = models.ForeignKey(Quest, on_delete=models.CASCADE)
    type = models.CharField(
        max_length=20, choices=QuestionType.choices, default=QuestionType.SHORT_ANSWER
    )
...
```